examples/keras_recipes/tf_serving.py

```diff
@@ -0,0 +1,445 @@
+
+From the TensorFlow Serving [GitHub page](https://github.com/tensorflow/serving):
+>TensorFlow Serving is a flexible, high-performance serving system for machine learning models, designed for production environments. It deals with the inference aspect of machine learning, taking models after training and managing their lifetimes, providing clients with versioned access via a high-performance, reference-counted lookup table. TensorFlow Serving provides out-of-the-box integration with TensorFlow models, but can be easily extended to serve other types of models and data."
+
+To note a few features:
+
+- Can serve multiple models, or multiple versions of the same model simultaneously
+- Exposes both gRPC as well as HTTP inference endpoints
+- Allows deployment of new model versions without changing any client code
+- Supports canarying new versions and A/B testing experimental models
+- Adds minimal latency to inference time due to efficient, low-overhead implementation
+- Features a scheduler that groups individual inference requests into batches for joint execution on GPU, with configurable latency controls
+- Supports many servables: Tensorflow models, embeddings, vocabularies, feature transformations and even non-Tensorflow-based machine learning models
+
+
+This guide creates a simple [MobileNet](https://arxiv.org/abs/1704.04861) model using the [Keras applications API](https://keras.io/api/applications/), and then serves it with [TensorFlow Serving](https://www.tensorflow.org/tfx/guide/serving). The focus is on TensorFlow Serving, rather than the modeling and training in TensorFlow.
+
+> Note: you can find a Colab notebook with the full working code at [this link](https://colab.research.google.com/drive/1nwuIJa4so1XzYU0ngq8tX_-SGTO295Mu?usp=sharing)
+
+# Dependencies
+"""
+
+import os
+import json
+import shutil
+import tempfile
+import requests
+import numpy as np
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+"""
+# Model
+
+Here we will load a pre-trained [MobileNet](https://arxiv.org/abs/1704.04861) from the [Keras applications](https://keras.io/api/applications/), this is the model that we are going to serve.
+"""
+
+model = tf.keras.applications.MobileNet()
+
+"""
+# Preprocessing
+
+Most models don't work out of the box on raw data, they usually require some kind of preprocessing step to adjust the data to the model requirements, in the case of this MobileNet we can see from its [API page](https://keras.io/api/applications/mobilenet/) that it requires three basic steps for its input images:
+
+- Pixel values normalized to the `[0, 1]` range
+- Pixel values scaled to the `[-1, 1]` range
+- Images with the shape of `(224, 224, 3)` meaning `(height, width, channels)`
+
+We can do all of that with the following function:
+"""
+
+
+def preprocess(image, mean=0.5, std=0.5, shape=(224, 224)):
```

Comment:
For a Keras application, you should use `tf.keras.applications.mobile_net.preprocess_input`